packages/next-admin/src/components/DataTable.tsx

```diff
@@ -155,11 +155,7 @@ export function DataTable({
                 data-state={row.getIsSelected() && "selected"}
                 className="cursor-pointer hover:bg-indigo-50"
                 onClick={() => {
-                  router.push({
-                    pathname: `${basePath}/${resource.toLowerCase()}/${
-                      row.original[modelIdProperty].value
-                    }`,
-                  });
+                  window.location.href = `${basePath}/${resource.toLowerCase()}/${row.original[modelIdProperty].value}`
```

Comment:
Why ?

Reply:
Route handle by next with `next/router` doesn't allow us to catch event `beforeunload`.